src/extension/background-script/actions/setup/setIcon.ts

```diff
@@ -0,0 +1,59 @@
+import browser, { Runtime } from "webextension-polyfill";
+import { SetIconMessage } from "~/types";
```

Comment:
Could we rename this to `MessageSetIcon`? So far all message-types start with `Message`